app/models/html_document.rb

```diff
@@ -41,10 +41,21 @@ def parse_content
     # it to html, then running it through Loofah again, it's because Loofah's 'to_text' method is
     # much smarter about whitespace than Nokogiri, but you can't run `to_text` on the Nokogiri
     # elements extracted by "html.at('main')". Hence the jumping through hoops.
-    plain_text = Loofah.fragment(main_html).scrub!(:whitewash).to_text(encode_special_chars: false)
+    plain_text = Loofah.fragment(main_html).
+      scrub!(tag_scrubber).
+      scrub!(:whitewash).
+      to_text(encode_special_chars: false)
     plain_text.gsub(/[ \t]+/,' ' ).gsub(/[\n\r]+/, "\n").chomp.lstrip
   end
```

Comment:
what if plain_text is nil?

Reply:
It won't ever be nil. Even if `main_html` is nil, `to_text` will convert that to an empty string:
```ruby
Loofah.fragment(nil).to_text
#=> ""
```